modules/@apostrophecms/admin-bar/ui/apos/components/TheAposContextModeAndSettings.vue

```diff
@@ -125,6 +125,9 @@ export default {
         }
       }
     },
+    isAutopublished() {
+      return this.context._aposAutopublish || window.apos.modules[this.context.type].autopublish || false;
```

Comment:
If `_aposAutopublish` is defined it should be respected regardless of whether it is true or false.